sos/report/plugins/microshift_ovn.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2023 Pablo Acevedo <pacevedo@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class MicroShiftOVN(Plugin, RedHatPlugin):
+    """This plugin is used to collect MicroShift 4.x OVN logs.
+    """
+    short_desc = 'MicroShift OVN'
+    plugin_name = "microshift_ovn"
+    profiles = ('microshift',)
+
+    def setup(self):
+        self.add_copy_spec([
+            '/etc/openvswitch/conf.db',
+            '/etc/openvswitch/default.conf',
+            '/etc/openvswitch/system-id.conf'])
+
+        _ovs_cmd = 'ovs-appctl -t /var/run/ovn/'
+        _subcmds = [
+            'coverage/show',
+            'memory/show',
+            'ovsdb-server/sync-status'
+        ]
+        for file, db in [
+            ('ovnnb_db.ctl', 'OVN_Northbound'),
+            ('ovnsb_db.ctl', 'OVN_Southbound')]:
+            self.add_cmd_output(
+                [f"{_ovs_cmd}{file} {cmd}" for cmd in _subcmds])
```

Comment:
flake8 is failing here due to the indention on line 35. Can either indent one more level, or do something like below. Up to you.

```python
for file, db in [('ovnnb_db.ctl', 'OVN_Northbound'),
                 ('ovnsb_db.ctl', 'OVN_Southbound')]:
```

Reply:
Done